tests/publiccloud/migration.pm

```diff
@@ -19,60 +20,142 @@ use strict;
 use warnings;
 use utils;
 use publiccloud::utils;
+use File::Basename;
+use Data::Dumper;
 
 our $target_version = get_required_var('TARGET_VERSION');
-our $not_clean_vm = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
+our $not_clean_vm   = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
 
 sub run {
-    my ($self, $args) = @_;
+    my ( $self, $args ) = @_;
+    #Download unpublished DMS package and install rpm
+    my $dms_rpm_url          = get_var('PUBLIC_CLOUD_DMS_RPM_URL');
+    my $dms_rpm     = basename($dms_rpm_url);
+    my $repo_key_url = get_var('PUBLIC_CLOUD_DMS_REPO_KEY_URL');
+    my $repo_key     = basename($repo_key_url);
+    my $pc_rpm           = get_var('PUBLIC_CLOUD_DMS_PC_RPM');
+    my $pc_act_location       = get_var('PUBLIC_CLOUD_PC_ACT_LOCATION');
+    my $act_rpm          = get_var('PUBLIC_CLOUD_DMS_ACT_RPM');
+    record_info('DEBUG', $pc_rpm);
```

Comment:
ah this was leftovers which I was using to debug , please remove all `record_info`'s which has label DEBUG . sorry that I haven't cleaned up the code before show it to you